docs/docs/networks.md

```diff
@@ -4,111 +4,112 @@
+| Chain ID       | fetchhub-4                                                                                                                                                                                                                 |
+| Block range    | 5,300,201 -->                                                                                                                                                                                                              |
+| Date range     | 05/04/2022 -->                                                                                                                                                                                                             |
+| Denomination   | afet                                                                                                                                                                                                                       |
+| Decimals       | 18 (1fet = 1000000000000000000afet)                                                                                                                                                                                        |
+| Version        | [v0.10.2](https://github.com/fetchai/fetchd/releases/tag/v0.10.2)                                                                                                                                                          |
+| RPC Endpoint   | <https://rpc-fetchhub.fetch.ai:443>                                                                                                                                                                                        |
+| GRPC Endpoint  | <https://grpc-fetchhub.fetch.ai:443>                                                                                                                                                                                       |
+| REST Endpoint  | <https://rest-fetchhub.fetch.ai:443>                                                                                                                                                                                       |
+| Block Explorer | [https://explore-fetchhub.fetch.ai](https://explore-fetchhub.fetch.ai)                                                                                                                                                     |
+| Token Faucet   | N/A                                                                                                                                                                                                                        |
+| Genesis        | `curl https://raw.githubusercontent.com/fetchai/genesis-fetchhub/main/fetchhub-4/data/genesis_migrated_5300200.json --output ~/.fetchd/config/genesis.json`                                                                |
```

Comment:
We already have genesis_chunked: `https://rpc-fetchhub.fetch.ai/genesis_chunked?chunk=0`. But using this repo is fine I guess - just created a tag and linked it instead of main branch, so it don't break on next change

```suggestion
| Genesis        | `curl https://raw.githubusercontent.com/fetchai/genesis-fetchhub/fetchhub-4/fetchhub-4/data/genesis_migrated_5300200.json --output ~/.fetchd/config/genesis.json`                                                                |
```